### DISCUSSION TITLE
Hardverváltoztatás felhőalapú pénztárgép esetén

### DISCUSSION THREAD

**Discussion by wlln22** [nav-gov-hu/eRECEIPT#42] [Kérdések és válaszok]
_”hardverváltoztatás alatt a hardveralapú e-pénztárgép engedélyezett perifériáinak változását kell érteni”_ 

Viszont az igazgatási szolgáltatási díjak felsorolása között a felhőalapú pénztárgépnél szerepel hardverváltoztatás díjtétel is.  Felhőalapú e-pénztárgép esetén a hardverváltoztatás mire vonatkozik?

**Comment by BatariZ** [comment]
Az e-pénztárgép rendelet 15. § (1) bekezdés b) pontja, valamint (2) bekezdés b) pontja alapján a forgalmazási engedély módosítása iránti kérelemmel kezdeményezhető az e-pénztárgép hardverének megváltoztatása. Hardverváltoztatás alatt a hardveralapú e-pénztárgép engedélyezett perifériáinak változását kell érteni.

**Reply by wlln22** [reply]
Ugyan a rendelet értelmezése alapján hardverváltoztatás alatt a hardveralapú e-pénztárgépek engedélyezett perifériáinak módosítását kell érteni, a rendelet E) 3. pontjában szereplő alábbi szöveg mégis ellentmondásosnak tűnik ezzel kapcsolatban:

„A kezelő által meghatározott eszközre telepített alkalmazás kizárólag az engedélyben meghatározott perifériákkal kommunikálhat, a kérelemhez mellékelt dokumentáció szerinti módon.”

A rendelet ezen pontja hogyan értelmezendő?
